rust/abacus-base/src/contract_sync/metrics.rs

```diff
@@ -5,15 +5,31 @@ use std::sync::Arc;
 /// Struct encapsulating prometheus metrics used by the ContractSync.
 #[derive(Debug, Clone)]
 pub struct ContractSyncMetrics {
-    /// Most recently indexed block height (label values differentiate checkpoints
-    /// vs. messages)
+    /// Most recently indexed block height (label values differentiate
+    /// checkpoints vs. messages)
+    ///
+    /// Labels:
+    /// - `data_type`: which indexer is recording the information.
```

Comment:
What are the different indexers here? Isn't it just checkpoint vs messages?

Reply:
InterchainGasPaymasterIndexer or OutboxIndexer. Generically speaking this is for "ContractSyncMetrics" so it will be recording for whatever the indexer is syncing. The actual string values right now are either "messages" or "gas_payments".

Reply:
Gotcha, just minor then, but if I read the description, i would have expected values like `OutboxIndexer`, when in fact it is `messages`

Reply:
I am going to resolve this to get things merged since this does not feel like it really relates to the documentation itself. If we want to change the string values I would be down for that as part of a separate PR. Please make a ticket if that is desirable.